sequencer/src/header.rs

```diff
@@ -600,6 +585,56 @@ mod test_headers {
         .run()
     }
 
+    #[test]
+    fn test_new_header_deposits_one() {
+        TestCase {
+            l1_deposits: vec![FeeInfo::new(Address::default(), 1)],
+            ..Default::default()
+        }
+        .run()
+    }
+
+    #[test]
+    fn test_new_header_deposits_many() {
+        TestCase {
+            l1_deposits: [
+                (Address::default(), 1),
+                (Address::default(), 2),
+                (Address::random(), 3),
+            ]
+            .iter()
+            .map(|(address, amount)| FeeInfo::new(*address, *amount))
```

Comment:
I feel like this could become a common pattern. Does it warrant implementing `From<(Address, u64) for FeeInfo`?

Reply:
Was also wondering. It might. I think we can add it later if we really need it.